app-containers/kubectl/autobuild/build

```diff
@@ -0,0 +1,13 @@
+abinfo "Build kubectl ..."
+make kubectl
+
+abinfo "Install kubectl ..."
+install -Dvm755 ./_output/bin/kubectl -t "$PKGDIR"/usr/bin
+
+
+abinfo "Installing bash and zsh completions ..."
+
+"$PKGDIR"/usr/bin/kubectl completion bash | \
+	install -Dvm644 /dev/stdin "$PKGDIR"/usr/share/bash-completion/completions/kubectl
```

Comment:
This is such a wild way to install completions - what does `kubectl completion bash` output?

Reply:
The `kubectl completion bash` command outputs Bash completion scripts tailored for `kubectl` commands, enhancing command-line usability.